dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcEventAnalyticsTableManager.java

```diff
@@ -167,6 +132,7 @@ public JdbcEventAnalyticsTableManager(
         analyticsExportSettings,
         periodDataProvider,
         sqlBuilder);
+    FIXED_COLS = EventAnalyticsColumn.getColumns(sqlBuilder);
```

Comment:
This column is not final, but static and in upper-case.

It is better to make it non-static and in lower-case, as it is now a regular singleton List at the class scope.